pkg/Apps/Database/Library/DbApp.js

```diff
@@ -6,10 +6,8 @@
  * license that can be found in the LICENSE file or at
  * https://developers.google.com/open-source/licenses/bsd
  */
```

Comment:
Another goal is to convert DbApp to CommonApp+Graph format. If we have a habit of using GraphMaker objects in App Graphs, there is a virtuous cycle in favor of NodeTypes.